### ISSUE TITLE
Erlang compatibility

### ISSUE THREAD

**Issue by joshprice** [joshprice/graphql-elixir#78]
Make it possible and easy to call GraphQL Elixir from pure Erlang (and LFE, etc).

**Comment by joshprice** [comment]
Add rebar3 support so that Elixir/Mix isn't required

**Comment by marutha** [comment]
+1 any updates on this or sample how we can define and call in pure Erlang?

**Comment by joshprice** [comment]
The standard way to do this is described here: http://elixir-lang.org/crash-course.html#adding-elixir-to-existing-erlang-programs

Can you try it out and report back any issues you find?

**Comment by marutha** [comment]
Yes, I did give a try on this and did succeed to a good level, but downside is we have an dependency that we have to define the schema  and resolve functions in elixir and and compile them in elixir and then only we can call them in Erlang.

Also we have to ensure that 'Elixir' lib directory is added to "lib" of Erlang.
